README.md

```diff
@@ -107,7 +107,7 @@ accomplish this as follows:
     2.  The `Sec-CH-UA-Platform` header field represents the platform's brand and major version. For example:
 
         ```http
-        Sec-CH-UA-Platform: "Win 10"
+        Sec-CH-UA-Platform: "Win; v=10"
```

Comment:
Do any platforms reveal more-than-significant/non-integer versions to the browser?